code/modules/mob/living/carbon/alien/alien_defense.dm

```diff
@@ -38,7 +38,8 @@ In all, this is a lot like the monkey code. /N
 				M.do_attack_animation(src, ATTACK_EFFECT_BITE)
 				playsound(loc, 'sound/weapons/bite.ogg', 50, TRUE, -1)
 				visible_message("<span class='danger'>[M.name] bites [src]!</span>", \
-						"<span class='userdanger'>[M.name] bites you!</span>", null, COMBAT_MESSAGE_RANGE)
+								"<span class='userdanger'>[M.name] bites you!</span>", null, COMBAT_MESSAGE_RANGE, M)

```

Comment:
Btw sometimes there's [M.name] used in the messages, but usually it's just [M].
Should [M.name] be changed to [M], or vice versa, or does it matter?

Reply:
M and M.name should be the same, it doesn't pull real_name or anything like that.

Reply:
Is there any reason for this to be changed to just [M]? And if so, should all [M]'s in visible_messages be changed to [M.name]? Which one would be better practice in visible_messages?

Reply:
[M]